WcaOnRails/app/webpacker/components/Competitions/MyCompetitions/index.jsx

```diff
@@ -0,0 +1,310 @@
+import React, {useState} from 'react';
+import {
+  Accordion,
+  Header,
+  Icon,
+  Table,
+  TableBody,
+  TableHeader,
+  Popup,
+  Checkbox,
+} from 'semantic-ui-react';
+import I18n from '../../../lib/i18n';
+import {
+  competitionReportEditUrl,
+  competitionReportUrl,
+  myCompetitionsAPIUrl,
+  personUrl,
+  meAPIUrl,
+  permissionsAPIUrl,
+  editCompetitionsUrl,
+  competitionRegistrationsUrl,
+} from '../../../lib/requests/routes.js.erb';
+import useLoadedData from '../../../lib/hooks/useLoadedData';
+import Loading from '../../Requests/Loading';
+import { countries } from '../../../lib/wca-data.js.erb';
+
+const defaultPermissions = {
+  can_attend_competitions: { scope: [] },
+  can_organize_competitions: { scope: [] },
+  can_administer_competitions: { scope: [] },
+};
+
+const dateRange = (startDate, endDate) => {
```

Comment:
Can you move utils outside of this file into a (probably already established) utils file/directory